app/assets/javascripts/serviceworker-companion.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
 
-if ('serviceWorker' in navigator) {
+if (
+  'serviceWorker' in navigator &&
+  !(
+    /Safari/i.test(navigator.userAgent) &&
```

Comment:
It looks like you can have Safari on windows(😱), should we just do the Safari check and leave off the apple check here?

Reply:
@mstruve Apple hasn't released Safari on Windows since Safari 6.x, which was probably almost a decade ago at this point, and it didn't support `ServiceWorker`. Shouldn't need to worry about it. :-)

Reply:
> Do we know if not trying to load a SW at all will stop this or will it continue to use the existing installed SW?

I'm not sure TBH, will defer to ServiceWorker experts like @benhalpern